unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/MeshesInfo/Rendereable.cs

```diff
@@ -31,14 +31,22 @@ public class Rendereable : ICloneable
         // This can be safely removed once we get rid of the old importer
         public bool isGLTFast = false;
 
+        public static EventHandler<Rendereable> OnRendereableAdded;
```

Comment:
Please avoid static. Our general approach to it is using `DataStore`

Reply:
Solved using `DataStore`